brainstorm/02-event-powered-judges.md

```
@@ -126,8 +126,84 @@ No decision needed yet. These are future judge patterns that become possible onc
+  - name: my-custom-judge
+    type: check
+    script: judges/custom.py
+```
+
+The harness resolves `type: builtin` to bundled Python scripts. Simple to use, but couples the judge library to the harness release cycle.
+
+**Option 2: Judge pack directory**
+
+Ship a `judges/` directory in the harness with categorized judge scripts:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)